ndc-models/src/relational_query/mod.rs

```diff
@@ -16,6 +16,7 @@ use crate::{ArgumentName, CollectionName, FieldName, OrderDirection};
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 #[schemars(title = "RelationalQuery")]
+#[skip_serializing_none]
```

Comment:
This works, but I honestly prefer omitting the Option like we do elsewhere:

```rs
  // this goes on the property not the struct
  #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
```

Reply:
Yeah, that's what I'm used to, but this is how we do it everywhere else in this repo currently.